src/main/kotlin/dev/stalla/model/HrefOnlyImage.kt

```diff
@@ -6,7 +6,12 @@ import dev.stalla.builder.validating.ValidatingHrefOnlyImageBuilder
 /**
  * An `<image href="...">` tag. The `href` attribute is _mandatory_.
  *
+ * Use the [builder][HrefOnlyImage.Factory.builder] method to obtain an
```

Comment:
Good spot. The difference is actually intended and present in the four models that only take a single parameter (`HrefOnlyImage`, `Bitlove`, `Content`, `Fyyd`), since its not really possible to mess up the order of arguments there. But for consistency, we can have the comments discourage instantiation in Java all the time, regardless of the number of arguments.

Reply:
Ok, seems I have still been inconsistent nonetheless. `EpisodePodlove` also only takes a single parameter, but it has the other comment...